src/commands/rank.ts

```diff
@@ -63,7 +63,7 @@ const RankCommand: SlashLevel.ICommand = {
 
 		const url = `${
 			CONFIG.IMAGE_API_URL
-		}/v2/canvas/rankcard?color=${encodeURIComponent(
+		}/?color=${encodeURIComponent(
```

Comment:
TL;DR: Remove API and try to send images directly.

I'm asking because I don't know exacly if its going to work. Can we send images directly with interactions? I mean some real buffers instead of putting image links. If we can, then we can change the url with real image buffer and remove the api thing completely.